README.md

```diff
@@ -1 +1,154 @@
-# ros-frontend
\ No newline at end of file
+# Resource Optimization Service (Frontend)
+
+Resource Optimization is a service aimed to provide existing Red Hat Enterprise Linux customers with an ability to grasp control over their investment in the Public Cloud. Read more about it [here](https://www.redhat.com/en/blog/optimize-public-cloud-workloads-rhel-red-hat-insights-resource-optimization)
+
+# Prerequisites
+
+### Install
+
+- [Node](https://nodejs.org/en/download/)
+- [Docker](https://docs.docker.com/get-docker/)
+
+
+# Setup Development Environment
+
+1. To get started with Resource Optimization Service (ROS), make sure you meet the prerequisites mentioned in both ros frontend and backend repos
```

Comment:
```suggestion
1. Make sure you meet the prerequisites mentioned in both frontend and backend repositories
```

I understand that repos is a short form of repositories. In my opinion here it's good to mention full word i.e `repositories`